cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -208,14 +210,28 @@ private Reference buildReference() {
             = new PublicationInstanceBuilderImpl(cristinObject);
         PublicationInstance<? extends Pages> publicationInstance
             = publicationInstanceBuilderImpl.build();
-        PublicationContext publicationContext = attempt(this::buildPublicationContext).orElseThrow();
+        PublicationContext publicationContext = attempt(this::buildPublicationContext)
```

Comment:
I was thinking, that the whole functionality of building a Reference could be moved to a new class ReferenceBuilder, but this is out of scope of this PR